fetch_teleop/scripts/gripper_keyboard.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python
+
```

Comment:
this needs a copyright header with License that matches the package license (copyright can be different, but the license should match that specified in the package.xml)